gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -771,3 +782,15 @@ ag_abgaben_geringf:
     ges_krankenv: 0.13
     ges_rentenv: 0.15
     st: 0.02
+
+ges_pflegev_zusatz_kinderlos:
+  name:
+    de: Altersgrenze Beitragszuschlag für Kinderlose
+    en: Age limit additional contribution for childless
+  description:
+    de: Nach Vollendung des 23. Lebensjahres erhöht sich der Beitragssatz für Kinderlose um 0.35 Beitragssatzpunkte
+    en: After completing the 23rd year of life the contribution rate for childless individuals is increased by 0.35 contribution rate points
+  unit: Years
+  reference: § 55 Abs. 3 SGB XI
+  1984-01-01:
+    altersgrenze: 22
```

Comment:
Rather use 23 and adjust the formula accordingly. Try to stay as close as possible to law.